### PR TITLE
GCS_MAVLink: Rate limit blocking accel calibrations

### PR DESCRIPTION
**Pull request by WickedShell**
This allows us to drop any queued commands that may have arrived while we were calibrating. Rather then entering a second and unexpected calibration.

EDIT: Testing was done against mission planner with a cube orange over usb and using the level accel cal button (the param5==2 path). I don't have a GCS here that lets me test the simple_accel_cal path. If I sent commands within 5 seconds of it finishing it was rejected, if I waited they worked again.

EDIT2: `accelcalsimple` in mavproxy looks like it works as intended.

### CODE REVIEW

File: libraries/GCS_MAVLink/GCS_Common.cpp [hunk 1]
```diff
@@ -4130,21 +4130,35 @@ MAV_RESULT GCS_MAVLINK::_handle_command_preflight_calibration(const mavlink_comm
...
             return MAV_RESULT_FAILED;
         }
         Vector3f trim_rad = AP::ahrs().get_trim();
         if (!AP::ins().calibrate_trim(trim_rad)) {
+            last_accel_cal_ms = AP_HAL::millis();
             return MAV_RESULT_FAILED;
         }
         // reset ahrs's trim to suggested values from calibration routine
         AP::ahrs().set_trim(trim_rad);
+            last_accel_cal_ms = AP_HAL::millis();
```
Comment (peterbarker) [inline libraries/GCS_MAVLink/GCS_Common.cpp:4151]: Indenting
```suggestion
        last_accel_cal_ms = AP_HAL::millis();
```

**Review by peterbarker — CHANGES_REQUESTED** [review]
LGTM apart from whitespace.  Specify testing done in PR, please.

File: libraries/GCS_MAVLink/GCS_Common.cpp (hunk 1 above)
Comment (WickedShell) [reply libraries/GCS_MAVLink/GCS_Common.cpp]: Fixed, sorry.

**Review by tridge — APPROVED** [review]

**Review by peterbarker — CHANGES_REQUESTED** [review]
Needs to pass CI, but otherwise acceptable.

**Pull request merged**